<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COMDeviceBase</class>
 <widget class="QWidget" name="COMDeviceBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="70,30">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>receive</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_device_rec_format"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit_device_rec">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_device_rec_clear">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>send</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_device_send_format"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_device_send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit_device_send"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>format</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
